exporter/clickhousetracesexporter/clickhouse_exporter_v3.go

```diff
@@ -335,7 +347,11 @@ func (s *storage) pushTraceDataV3(ctx context.Context, td ptrace.Traces) error {
 					}
 					batchOfSpans = append(batchOfSpans, structuredSpan)
 
-					serializedStructuredSpan, _ := json.Marshal(structuredSpan)
+					serializedStructuredSpan, err := json.Marshal(structuredSpan)
+					if err != nil {
+						zap.S().Error("Error in marshalling structuredSpan: ", err)
+						return err
```

Comment:
A general rule is return or log, not both https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully as it will log errors twice (one here and one at the export helper caller)